### PR TITLE
Img

### PR DESCRIPTION
**Pull request by codingoak**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,30 +19,44 @@ <h1>Daniel Eicher</h1>
...
+    </ul>
+  </section>
+  <section>
+    <h2>Hobbies</h2>
+    <ul>
+      <li>Hiking</li>
+      <li>Coding</li>
+      <li>Reading books</li>
+    </ul>
+  </section>
```
Comment (teifler) [inline index.html:40]: Well done that code looks good. Dont forget to create new branches because this is not related to the branch name anymore (img).

Comment (MichiBerger) [reply index.html:40]: Very nice ;)Just one thing... I would recommend you to set an id to the `section` tags

File: index.html [hunk 2]
```diff
@@ -19,30 +19,44 @@ <h1>Daniel Eicher</h1>
...
   <section id="contact">
     <h2>Contact</h2>
     <ul>
       <li><b>Phone: </b>+34 642 104 722</li>
-      <li><b>Mail: </b>danieleicher@gmx.net</li>
+      <li><b>Mail: </b><a href="mailto:danieleicher@gmx.net">danieleicher@gmx.net</a></li>
+      <li><b>Connect with me on:<br><b>
```
Comment (MichiBerger) [inline index.html:46]: Cool list and a good idea with the pictures 👍  Just one question: what is your intention with the `b` tag? If you want to have the text bold and you want to use it more then once in your code a `span` tag with a class might be the better choice ;) good work so far

Comment (codingoak) [reply index.html:46]: Thank you. Yes - later I will create a bold class and style it with css :)

File: index.html (hunk 1 above)
Comment (codingoak) [reply index.html:40]: yes i fixed it, thx.
